minio.json

```
@@ -1,9 +1,9 @@
 {
     "MinIO": {
-        "description": "MinIO is an S3-compatible object storage server. This rock-on requires Rockstor 3.9.2-39 or later and is compatible with x86_64/amd64 machines only.<p>Based on the official MinIO docker image: <a href='https://hub.docker.com/r/minio/minio' target='_blank'>https://hub.docker.com/r/minio/minio</a>.",
-        "version": "1.0",
+        "description": "MinIO is an S3-compatible object storage server.",
```

```suggestion
        "description": "MinIO is an S3-compatible object storage server. <p>Based on the official MinIO docker image: <a href='https://hub.docker.com/r/minio/minio' target='_blank'>https://hub.docker.com/r/minio/minio</a>, available for x86-64/amd64 and arm64.",
```

@MikeMcPherson The venerable @FroggyFlox would want us to pop back in the image used and the architectures. We really need to formalise this into our Rock-on definition but for now this serve as a stop gap and lets folks see if they can run on the Pi4 for example.